lib/utils/documentation-urls.js

```diff
@@ -1,4 +1,6 @@
-const baseUrl = 'https://adopted-ember-addons.github.io/ember-electron/docs/';
+const { homepage } = require('../../package.json');
```

Comment:
How does this work? The homepage should be a field in the package.json, not the entire package.json, right? I'm also not sure what this documentation urls util does? Do we need it since we have addon docs?

Reply:
The `{ ... }` does [destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment)

Reply:
Ohhhh, wow you are right. It is early 😂. I still wonder if we need these doc utils at all? I would think not with addon docs?

Reply:
Maybe, but I think some are used in error/warning messages, which I would say has value.

Reply:
Ah okay